[Excel] (Events) (Worksheets) Add onFormulaChanged conceptual docs (#2726)
OfficeDev/office-js-docs-pr: docs/excel/excel-add-ins-worksheets.md

}
```

## Detect formula changes (preview)

> [!NOTE]
> The `Worksheet.onFormulaChanged` event is currently only available in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
> 

> Your add-in can track changes to formulas in a worksheet. This may be useful when a worksheet is connected to an external database and formula changes in the worksheet need to trigger corresponding updates in the external database.

Reviewer: A couple suggestions to make this sound more scenario-driven.
```suggestion
Your add-in can track changes to formulas in a worksheet. This is useful when a worksheet is connected to an external database. When the formula changes in the worksheet, the event in this scenario triggers corresponding updates in the external database.
```

Author: Thank you!

> To detect changes to formulas, [register an event handler](excel-add-ins-events.md#register-an-event-handler) for the [onFormulaChanged](/javascript/api/excel/excel.worksheet#onFormulaChanged) event of a worksheet.  Event handlers for the `onFormulaChanged` event receive a [WorksheetFormulaChangedEventArgs](/javascript/api/excel/excel.worksheetformulachangedeventargs) object when the event fires.

Reviewer: Extra space.
```suggestion
To detect changes to formulas, [register an event handler](excel-add-ins-events.md#register-an-event-handler) for the [onFormulaChanged](/javascript/api/excel/excel.worksheet#onFormulaChanged) event of a worksheet. Event handlers for the `onFormulaChanged` event receive a [WorksheetFormulaChangedEventArgs](/javascript/api/excel/excel.worksheetformulachangedeventargs) object when the event fires.
```
